<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1303.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>781</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>控制界面</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1202.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_temp">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1203.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hum">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1204.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ill">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>60</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1205.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_smoke">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1206.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_co2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>140</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1209.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_gas">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>140</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1210.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ap">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1207.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pm">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1208.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>220</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1211.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_air">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>220</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1212.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_warning">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>220</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1213.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_curtain">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>220</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1218.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_rfid">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1215.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_lamp">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1216.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_dvd">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1214.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_fan">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1217.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tv">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>310</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1219.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget_4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>湿度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_hum">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>80</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>光照：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_ill">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>80</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>烟雾：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_smoke">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_7">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>气压：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_air">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_8">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>160</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PM2.5：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_pm">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>160</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Co2：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_co2">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_10">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>160</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>燃气：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_gas">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_11">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>113</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>人体红外：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_inf">
        <property name="text">
         <string>无人</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>温度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lab_temp">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>模式界面</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1202.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_ld">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>联动模式</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ld1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>光照</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>温度</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PM2.5</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="ld2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>&lt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&gt;=</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="ld3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>90</y>
       <width>113</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ld">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>108</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>联动模式开启</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>98</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>报警灯</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>98</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>风扇</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>140</y>
       <width>98</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>窗帘</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chk4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>140</y>
       <width>98</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>射灯</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_mode1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>旅游模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_mode3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>320</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>安防模式</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_mode2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>220</y>
       <width>117</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>夜间模式</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>智慧模式</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1202.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>631</width>
       <height>271</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>系统配置</string>
    </attribute>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/1202.jpg);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>波特率：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb2">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>140</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>校验位：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb3">
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>数据位：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb4">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="btn_serialopen">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>230</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_tcp">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>280</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>连接服务器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_server">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>330</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>监听端口</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>串口号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb1">
        <item>
         <property name="text">
          <string>ttyUSB0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ttySAC0</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
